lens_v2/schema.graphql

```diff
@@ -22,4 +22,325 @@ type DefaultProfileSet @entity {
+}
+
+type AirSocialProfileRenewalTransaction implements AirTransaction @entity {
+  id: ID! #<transactionHash>-<logOrCallIndex>-<tokenId>-<expiryTimestamp>
+  expiryTimestamp: BigInt!
+  renewalCost: BigInt!
+  profile: AirSocialProfile!
+  tokenId: String!
+  tokenAddress: AirToken!
+  from: AirAccount!
+  to: AirAccount!
+  transactionHash: String! 
+  logOrCallIndex: BigInt!
+  block: AirBlock!
+  index: BigInt!
+  protocolType: AirProtocolType!  #SOCIAL
+  protocolActionType: AirProtocolActionType!  #SOCIAL_PROFILE_NAME_RENEWAL
+}
+
+type AirSocialProfileRecoveryAddressChangeTransaction implements AirTransaction @entity {
+  id: ID! #<transactionHash-logOrCallIndex-tokenId>
+  oldRecoveryAddress: AirAccount
+  newRecoveryAddress: AirAccount!
+  profile: AirSocialProfile!
+  tokenId: String!
+  tokenAddress: AirToken!
+  from: AirAccount!
+  to: AirAccount!
+  transactionHash: String! 
+  logOrCallIndex: BigInt!
+  block: AirBlock!
+  index: BigInt!
+  protocolType: AirProtocolType!  #SOCIAL
+  protocolActionType: AirProtocolActionType!  #SOCIAL_PROFILE_RECOVERY_ADDRESS_CHANGE
+}
+
+type AirSocialUserHomeUrlChangeTransaction implements AirTransaction @entity {
+  id: ID! #<transactionHash-logOrCallIndex-tokenId>
+  oldHomeUrl: String
+  newHomeUrl: String!
+  user: AirSocialUser!
+  tokenId: String!
+  tokenAddress: AirToken!
+  from: AirAccount!
+  to: AirAccount!
+  transactionHash: String! 
+  logOrCallIndex: BigInt!
+  block: AirBlock!
+  index: BigInt!
+  protocolType: AirProtocolType!  #SOCIAL
+  protocolActionType: AirProtocolActionType!  #SOCIAL_USER_HOME_URL_CHANGE
+}
+
+type AirSocialUserRecoveryAddressChangeTransaction implements AirTransaction @entity {
+  id: ID! #<transactionHash-logOrCallIndex-tokenId>
+  oldRecoveryAddress: AirAccount
+  newRecoveryAddress: AirAccount!
+  user: AirSocialUser!
+  tokenId: String!
+  tokenAddress: AirToken!
+  from: AirAccount!
+  to: AirAccount!
+  transactionHash: String! 
+  logOrCallIndex: BigInt!
+  block: AirBlock!
+  index: BigInt!
+  protocolType: AirProtocolType!  #SOCIAL
+  protocolActionType: AirProtocolActionType!  #SOCIAL_USER_RECOVERY_ADDRESS_CHANGE
+}
+
+type AirSocialUserDefaultProfileChangeTransaction implements AirTransaction
+  @entity {
+  id: ID! #<transactionHash-logOrCallIndex-user.Id>
+  oldDefaultProfile: AirSocialProfile
+  newDefaultProfile: AirSocialProfile
+  user: AirSocialUser!
+  from: AirAccount!
+  to: AirAccount!
+  transactionHash: String!
+  logOrCallIndex: BigInt!
+  block: AirBlock!
+  index: BigInt!
+  protocolType: AirProtocolType! #SOCIAL
+  protocolActionType: AirProtocolActionType! #SOCIAL_USER_DEFAULT_PROFILE_CHANGE
+}
+
+type Approval @entity(immutable: true) {
+  id: Bytes!
+  owner: Bytes! # address
+  approved: Bytes! # address
+  tokenId: BigInt! # uint256
+  blockNumber: BigInt!
+  blockTimestamp: BigInt!
+  transactionHash: Bytes!
+}
+
+type ApprovalForAll @entity(immutable: true) {
```

Comment:
Is this because we are planning to track approvals henceforth?

Reply:
Similar comment